migrations/201802190000-CreateUserTable.js

```diff
@@ -5,16 +5,17 @@ module.exports = {
         return queryInterface.createTable('user',
             {
                id: {
-                    type: Sequelize.UUID,
+                    type: Sequelize.STRING(36), //Sequelize.UUID,
                     primaryKey: true,
-                    defaultValue: Sequelize.UUIDV4
+                    unique: true
+                    //defaultValue: Sequelize.UUIDV4
                 }, username: {
-                    type: Sequelize.STRING(64) + ' CHARSET utf8mb4 COLLATE utf8mb4_unicode_ci',
+                    type: Sequelize.STRING(64), // + ' CHARSET utf8mb4 COLLATE utf8mb4_unicode_ci',
```

Comment:
To my opinion the test should be done with dynamic insertion, instead from a dumped DB

Reply:
If you look to you dump file backup.sql, you'll see there's some columns in latin1 charset, and other in utf8, think it will be impossible to run your tests with the code of my repo with a database restored from this sql file
I tried to add dialect dependent code but doesn't help...